code/procs/jobprocs.dm

```diff
@@ -438,9 +438,9 @@
 					var/datum/abilityHolder/A = src.abilityHolder.deepCopy()
 					R.fields["abilities"] = A
 
-				R.fields["traits"] = list()
-				if(src.traitHolder && src.traitHolder.traits.len)
-					R.fields["traits"] = src.traitHolder.traits.Copy()
+				SPAWN_DBG(0)
+					if(src.traitHolder && src.traitHolder.traits.len)
+						R.fields["traits"] = src.traitHolder.traits.Copy()
```

Comment:
I assume this is because traitHolder isn't yet initialize at his point?

Reply:
It is initialized, however it will only contain traits that your character had assigned via the character creator, it will not include traits assigned by your job, which since only security gets these disks is security training and career alcoholic if HoS

Reply:
Either way spawn is a bandaid at best. But also I don't see how to do it bandaid-less.